ddp-workspace/projects/ddp-pancan/src/app/components/splash-pages/lms-page/lms-page.component.ts

```diff
@@ -41,6 +41,8 @@ export class LmsPageComponent implements OnInit {
 
     ngOnInit(): void {
         this.useSpecificLanguage('en');
+        // PEPPER-1327: If user opens /lms page, redirects url to lmsproject.og
```

Comment:
Probably don't need the ticket in the comment since the commit message covers that base.